cdap-examples/pom.xml

```diff
@@ -37,6 +37,7 @@
     <module>Purchase</module>
     <module>SparkKMeans</module>
     <module>SparkPageRank</module>
+    <module>LogAnalysis</module>
```

Comment:
Just wonder, how to you determine to insert into middle of the existing example modules but rather to the end?